src/backend/core/user/views.py

```diff
@@ -182,29 +182,34 @@ class PasswordAdminView(UpdateAPIView):
     View for admins to change users password.
     """
 
-    serializer_class = ChangePasswordSerializer
+    serializer_class = ChangePasswordSerializerAdmin
     permission_classes = (permissions.AllowAny,)
 
     def get_object(self, queryset=None):
-        obj = User.objects.get(email=self.kwargs["email"])
+        obj = User.objects.get(email=self.kwargs["id"])
         return obj
 
     @check_user_access_decorator({"user_change_permission"})
-    def update(self, request, *args, **kwargs):
-        self.object = self.get_object()
-        serializer = self.get_serializer(data=request.data)
-
-        if serializer.is_valid():
-            # set_password also hashes the password that the user will get
-            self.object.set_password(serializer.data.get("new_password"))
-            self.object.save()
-            response = {
-                "status": "success",
-                "code": "200",
-                "message": "Password updated successfully",
-                "data": [],
-            }
-
-            return Response(response)
-
-        return Response(serializer.errors, status=400)
+    def put(self, request, id):
+        try:
+            self.object = self.get_object()
+            serializer = self.get_serializer(data=request.data)
+
+            print("AFTER SERIALIZER", serializer)
+
+            if serializer.is_valid():
+                print("AFTER SERIALIZER VALID")
+                # set_password also hashes the password that the user will get
+                self.object.set_password(serializer.data.get("new_password"))
+                self.object.save()
+                response = {
+                    "status": "success",
+                    "code": "200",
```

Comment:
Do we want to send the code as object string in the response object? The 200 is default for `Response` but you can set it as `Response(response, status=status.HTTP_200_OK)` (see other parts of the code 😊 )

Reply:
Fixed.